conceptual/Npgsql/release-notes/8.0.md

```diff
@@ -52,11 +52,16 @@ For more information on registering Npgsql services in DI, see the documentation
 
 ## Other features
 
+* Allow using nullable value types with the generic `NpgsqlParameter<T>`, e.g. `NpgsqlParameter<int?>`.
+* Introduce a non-caching password provider callback via <xref:NpgsqlDataSourceBuilder.UsePasswordProvider?displayProperty=nameWithType>.
+* Allow customizing System.Text.Json JsonSerializationOptions via <xref:NpgsqlDataSourceBuilder.ConfigureJsonOptions?displayProperty=nameWithType>.
 * Improvements and cleanup for networking type mappings:
   * In addition to .NET <xref:System.Net.IPAddress>, PostgreSQL `inet` can also mapped to be mapped to <xref:NpgsqlTypes.NpgsqlInet>, which is an immutable struct containing both IP and netmask components.
   * PostgreSQL `cidr` is now mapped to the newly-introduced <xref:NpgsqlTypes.NpgsqlCidr>. The mapping to `ValueTuple<IPAddress, int>` has been removed.
+* Allow providing the root certificate programmatically via the new <xref:NpgsqlDataSourceBuilder.UseRootCertificate?displayProperty=nameWithType>
+* Add support for channel binding on encrypted connections, and added the `Channel Binding` connection string parameter to allow managing that.
```

Comment:
1. It was already there since at least npgsql 5.0, we just added an ability manage it
2. It doesn't look like we added `Channel Binding` connection string parameter to documentation

Reply:
> It was already there since at least npgsql 5.0, we just added an ability manage it

Ah my bad, I went through too quickly. I'll remove it then - that doesn't seem worth mentioning in this list.

> It doesn't look like we added Channel Binding connection string parameter to documentation

Added to this PR - but we need to doc various security-related things in the security page. I'll do that later.